commit 37523eb3fc0074f36427baee3bb8a0883290895e (refs/changes/62/3462/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-02-23T23:20:10+00:00 (1 year, 7 months ago)
    
    MORAY-460 Move Moray to new Triton origin image
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/Makefile b/Makefile
index 17c068c..3b5fd5e 100644
--- a/Makefile
+++ b/Makefile
@@ -44,9 +44,10 @@ CLEAN_FILES	+= node_modules cscope.files \
 #
 
 ifeq ($(shell uname -s),SunOS)
+	# Allow building on a SmartOS image other than sdc-*-multiarch 15.4.1.
+	NODE_PREBUILT_IMAGE	= 18b094b0-eb01-11e5-80c1-175dac7ddf02
 	NODE_PREBUILT_TAG	= zone
 	NODE_PREBUILT_VERSION	:= v0.10.48
-	NODE_PREBUILT_IMAGE = fd2cc906-8938-11e3-beab-4359c665ac99
 endif
 
 # RELENG-341: no npm cache is making builds unreliable
